spec/controllers/platform/api/v1/accounts_controller_spec.rb

```diff
@@ -42,29 +42,25 @@
         json_response = JSON.parse(response.body)
         expect(json_response['name']).to eq('Test Account')
         expect(json_response['locale']).to eq('es')
-        expect(json_response['enabled_features']).to eq(
-          {
-            'agent_management' => true
-          }
-        )
+        expect(json_response['enabled_features']['agent_management']).to be(true)
       end
 
       it 'creates an account with feature flags' do
         InstallationConfig.where(name: 'ACCOUNT_LEVEL_FEATURE_DEFAULTS').first_or_create!(value: [{ 'name' => 'inbox_management',
```

Comment:
```suggestion
        InstallationConfig.where(name: 'ACCOUNT_LEVEL_FEATURE_DEFAULTS').first_or_create!(value: [{ 'name' => 'disable_branding',
```
let's have this added to ensure that `disable_branding`: false in the params is working properly.

Reply:
I am done with this suggested changes.